src/library/components/signup/signup_functions.js

```diff
@@ -28,7 +28,7 @@ var renderSignupForm = function (properties, selectorOrElement) {
   ReactDOM.render(comp(properties), jQuery(selectorOrElement)[0])
 }
 
-var openModal = function (type, properties, closeFunc) {
+var openModal = function (type, properties = {}, closeFunc) {
   var modalContainer, modalContainerId, modalContainerSelector
```

Comment:
Default function arguments aren't supported by all browsers (e.g. no IE support). What are the browser requirements here?